CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+
+* Provide low effort way for authors to style core properties on browser built-in
+tooltips (for example, color properties, font properties, etc.)
+* This mechanism should handle accessibility, positioning, and input handling for
+the author by default.
+* Allow authors to override the default positioning behavior of the built-in
+tooltips.
+* Allow authors to change the timing of built-in tooltips.
+* Improve accessibility of the `title` attribute, particularly for keyboard users.
+* Guarentee security best practices, given that tooltips today can extend past
+the bounds of the browser Window.
+* Styled tooltips should produce consistent results across all platforms, ensuring
+an interoperable experience for authors.
+
+## Non-goals
+
+* This proposal is not meant to solve for tooltips with arbitrary HTML content, which
+can be covered by existing solutions, like `popover`, CSS Anchor positioning, and
+the not-yet-available `interesttarget` attribute.
+* Provide a means to set a pointer on the `title` based tooltip. The solution should
+be specified to allow for this functionality in the future, though.
+
+## User research
+
+Use cases in this explainer were collected from the discussion in issue
+[8930](https://github.com/w3c/csswg-drafts/issues/8930). Additional inspiration
+was drawn from discussions in issues
+[9447](https://github.com/w3c/csswg-drafts/issues/9447), and OpenUI issue
+[730](https://github.com/openui/open-ui/issues/730).
+
+There are also many articules on the web that walk authors through various
+options for creating custom tooltips using tools available to them today.
+One such [article](https://blog.replaybird.com/css-tooltip-examples/)
+walks through different options, along with various examples and use cases
+for tooltips on the web.
+
+## Existing and upcoming tools available for custom tooltips
+
+If an author would like to adjust the style of their tooltip today, they must resort
+to the one of the following alternative approaches to the built-in tooltip that they
+get with the `title` attribute.
+
+### Use an existing library or component
+
+Authors can use an existing library or component for a fully customizable tooltip.
+Some example libraries include:
+  - https://fluent2.microsoft.design/components/web/react/core/tooltip/usage
+  - https://www.opentip.org/
+  - https://floating-ui.com/?utm_source=popper.js.org
+  - https://www.npmjs.com/search?ranking=popularity&q=tooltip
+  - Many more   
+
+It is clear that there are many libraries and components for creating custom tooltips
+that authors can use to solve their need for more customization.
+
+If the library chosen if well architected to properly handle accessibility, this can
+also be a convenient way to get such functionality "for free". However, relying on
+external library to get accessibility right can be risky, which may limit the usability
+if not done correctly.
+
+The other downside of this approach is that it requires heavy use of JS, which can be
```

Comment:
Might want to spell out "Javascript", or just say "scripting"

Reply:
Done